codespell_lib/data/dictionary.txt

```diff
@@ -30914,6 +30949,7 @@ shif-tab->shift-tab
 shineing->shining
 shiped->shipped
 shiping->shipping
+shoft->shift
```

Comment:
Perhaps `short` too, although the most common fix does seem to be `shift` and by far?

Reply:
Since the "f" key is below the "r" key in qwerty, I think this typo could be quite common. So I've added that variant.